pkg/specgen/generate/config_linux.go

```diff
@@ -248,6 +248,13 @@ func addDevice(g *generate.Generator, device string) error {
 		}
 		g.Config.Mounts = append(g.Config.Mounts, devMnt)
 		return nil
+	} else if src == "/dev/fuse" {
+		// if the user is asking for fuse inside the container
+		// make sure the module is loaded.
+		f, err := unix.Open(src, unix.O_RDONLY|unix.O_NONBLOCK, 0)
+		if err != nil {
```

Comment:
Shouldn't this be 
if err == nil {
	unix.Close(f)
}

Reply:
yes!  Thanks for noticing it